docs/servo_power_test/src/config.ts

```diff
@@ -11,6 +11,7 @@ export class Config {
   private runner: TestRunner;
   private halt = true;
   private inProgress = false;
+  private configNum: number;
```

Comment:
Num -> Number

Reply:
I fixed this in #163, so please merge the PR ahead.